### DISCUSSION TITLE
Tevm quickstart feedback

### DISCUSSION THREAD

**Discussion by dionysuzx** [evmts/tevm-monorepo#966] [General]
I am going to use this as a working friction log for the Tevm quick start guide.

## Creating / installing
1. `npm create tevm my-app` didn't work. (`bun init && bun install tevm` worked fine though!)
    ```
    npm ERR! code E404
    npm ERR! 404 Not Found - GET https://registry.npmjs.org/create-tevm - Not found
    npm ERR! 404
    npm ERR! 404  'create-tevm@*' is not in this registry.
    ```
## Running the Ethereum JSON-RPC stuff
1. `Module '"tevm"' has no exported member 'EthGetBalanceRequest'.`: https://tevm.sh/getting-started/quick-start/#using-ethereum-json-rpc. 
2. `bun run index.ts` errored out:
    ```
    ❯ bun run index.ts
    613 |   jsonrpc: "2.0",
    614 |   method: req.method,
    615 |   result: numberToHex(
    616 |     await getBalanceHandler({ vm, forkUrl })({
    617 |       address: req.params[0],
    618 |       ...req.params[1].startsWith("0x") ? { blockNumber: BigInt(req.params[1]) } : {
                   ^
    TypeError: undefined is not an object (evaluating 'req.params[1].startsWith')
          at ~/fun/demo/tevm-app/node_modules/@tevm/procedures/dist/index.js:618:10
          at ~/fun/demo/tevm-app/node_modules/@tevm/procedures/dist/index.js:611:55
          at ~/fun/demo/tevm-app/node_modules/@tevm/procedures/dist/index.js:72:17
          at ~/fun/demo/tevm-app/index.ts:13:24
    ```

**Comment by roninjin10** [comment]
> npm create tevm my-app didn't work.

This one is not a bug this is an unreleased feature 
<img width="232" alt="image" src="https://github.com/evmts/tevm-monorepo/assets/35039927/ea1ecf10-0344-463d-8b26-861b1accd718">

> Module '"tevm"' has no exported member 'EthGetBalanceRequest'.: https://tevm.sh/getting-started/quick-start/#using-ethereum-json-rpc.

This is a mistake in the docs. This request type is imported from tevm/procedures-types and it's called EthGetBalanceJsonRpcRequest not EthGetBalanceRequest

> bun run index.ts

The docs had a bug in it and we could throw a nicer error message here
